packages/hdwallet-ledger/src/ethereum.ts

```diff
@@ -8,7 +8,7 @@ import EthereumTx from "ethereumjs-tx";
 // TODO: fix ts-ignore
 import * as ethereumUtil from "ethereumjs-util";
 import { arrayify, isBytes, joinSignature } from "ethers/lib/utils.js";
-import { isHexString } from "ethjs-util";
+import { isHexString } from "ethers/lib/utils";
```

Comment:
May have worked at some point because of transient deps but definitely isn't now
 
<img width="1221" alt="image" src="https://github.com/user-attachments/assets/4f4ce171-0b98-4f24-96a1-516daa1a6295">